test-kitchen/skills/omakase-off/SKILL.md

```
@@ -0,0 +1,402 @@
+```
+
+User can manually kill slow/stuck implementations.
+
+**When subagent reports complete:**
+- Use `code-review` to review the variant's implementation
+- Fix any Critical/Important issues before proceeding to Phase 4
+- Note: each variant reviewed independently
+
+### Phase 4: Evaluation
+
+**Step 1: Scenario testing (REQUIRED - not ad-hoc)**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)